src/sections/requirements.tex

```diff
@@ -1,99 +1,233 @@
+
+        Scenario: Benachrichtigung versenden\\
+        Given:  Benutzer ist vollständig angemeldet\\
+        And:    Mindestens 1 Empfänger ist konfiguriert\\
+        When:   Praxismitarbeiter tippt auf einen Benachrichtigungs-Button\\
+        Then:   Benachrichtigung wird an den zentralen Cloud Service gesendet\\
+        And:    Benachrichtitung wird an alle Mobile Clients versendet die sich für diese Benachrichtigung subscribed haben weitergeleitet\\
+        And:    Praxismitarbeiter erhält optische Rückmeldung, dass Benachrichtigung versendet wurde\\
+
+
+        Scenario: Keine Empfänger konfiguriert\\
+        Given:  Benutzer ist vollständig angemeldet\\
+        And:    Kein Empfänger ist konfiguriert\\
+        When:   Praxismitarbeiter tippt auf einen Benachrichtigungs-Button\\
+        Then:   Benachrichtigung wird an den zentralen Cloud Service gesendet\\
+        And:    Benachrichtitung wird nicht weitergeleitet\\
+
+    \subsubsection*{F02 - Benachrichtigungen Empfangen}
+
+        Scenario: Empfangen\\
+        Given:  Eine Benachrichtigung wurde von Mobile Client versendet\\
+        When:   Cloud Service Notification an Empfänger Mobile Client weiterleitet\\
+        Then:   Wird die Benachrichtigung vom Empfänger Mobile Client empfangen\\
+        And:    In einer Übersicht für empfangene Benachrichtigung angezeigt.\\
+
+
+    \subsubsection*{F03 - Fehlgeschlagene Benachrichtigungen}
+
+        Scenario: Fehler Rückmeldung\\
+        Given:  Eine Benachrichtigung wurde von Mobile Client versendet\\
+        When:   Weiterleitung von Cloud Service Notification an Empfänger schlägt auf Service Seite fehl\\
+        Then:   Der Praxismitarbeiter wird über den Fehler Informiert\\
+        And:    Der Praxismitarbeiter hat die Möglichkeit die Fehlgeschlagenen Benachrichtigungen zu wiederholen\\
+
+        Scenario: Confirm Retry\\
+        Given:  Benachrichtigung ist fehlgeschlagen\\
+        And:    Dialog zum wiederholen wird angezeigt\\
+        When:   Praxismitarbeiter bestätigt, dass wiederholt werden soll\\
+        Then:   Der Cloudservice versucht erneut, die fehlgeschlagenen zuzustellen\\
+
+        Scenario: Cancel Retry\\
+        Given:  Benachrichtigung ist fehlgeschlagen\\
+        And:    Dialog zum wiederholen wird angezeigt\\
+        When:   Praxismitarbeiter klick, dass nicht wiederholt werden soll\\
+        Then:   Werden die fehlgeschlagenen nicht wiederholt\\
+        And:    Zurück zur Notificationsansicht\\
+
+
+    \subsubsection*{F04 - Über Benachrichtigungen Notifizieren}
+
+        Scenario: Foreground\\
+        Given:  Mobile Client ist geöffnet\\
+        When:   Eine Benachrichtigung wird vom Mobile Client emfpangen\\
+        Then:   Ein Audio Signal erklingt\\
+        \\
+        Scenario: Background\\
+        Given:  Mobile Client läuft im Hintergrund\\
+        When:   Eine Benachrichtigung wird vom Mobile Client empfangen\\
+        Then:   Ein Audio Signal erklingt\\
```

Comment:
Nein tuts nicht, 
ausser wahrscheinlich wenn man dies in den Geräteeinstellungen unter den Push Meldungen konfiguriert. Aber hat dann nichts mit unserer Applikation zu tun

Reply:
Wie am telefon besprochen. 

Ich lass das drinn und wir dokumentieren, dass das über die Push Benachrichtigungs Konfiguration gemacht werden muss.